tests/failpoints/cases/test_merge.rs

```diff
@@ -893,8 +894,10 @@ fn test_node_merge_cascade_merge_with_apply_yield() {
     let r2 = pd_client.get_region(b"k5").unwrap();
     let r3 = pd_client.get_region(b"k9").unwrap();
 
+    assert_eq!(r1.get_id() % 4, 0);
+
     pd_client.must_merge(r2.get_id(), r1.get_id());
-    assert_eq!(r1.get_id(), 1000);
+    assert_eq!(r1.get_id() % 4, 0);
```

Comment:
I think maybe the original purpose of such case is to guarantee that target region won't be changed.

```suggestion
    assert_eq!(pd_client.get_region(b"k5").unwrap().get_id(), r1.get_id());
```